pages/quiz/about-you.tsx

```diff
@@ -78,6 +79,7 @@ const StyledLink = styled.a`
   justify-content: center;
   margin: auto;
   max-width: calc(100% - 2rem);
+  margin-bottom: 100px;
```

Comment:
Is this to account for the nav being in the way of the content? If it's for every page then it may be easier in the layout - but we can save for the responsive ticket as we'll want to toggle it off.